easybuild/easyconfigs/g/gtest/gtest-1.7.0-intel-2015a.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = "MakeCp"
+
+name = 'gtest'
```

Comment:
hmm, should we rename to googletest or GoogleTest, which seems to be the official name of the project?